polaris.shopify.com/src/components/TOC/TOC.tsx

```diff
@@ -102,25 +103,39 @@ function TOC({items}: Props) {
 
   useEffect(() => detectCurrentHeading(), [items]);
 
-  const Link = ({toId, linkText}: {toId: string; linkText: string}) => (
-    <a
-      href={`#${toId}`}
-      data-is-current={toId === idOfCurrentHeading}
-      onClick={(evt) => {
-        scrollIntoView(toId);
-        evt.preventDefault();
-      }}
-    >
-      {linkText}
-    </a>
-  );
+  const Link = ({toId, linkText}: {toId: string; linkText: string}) => {
+    const activeHeading = toId === idOfCurrentHeading;
+    return (
+      <Text
```

Comment:
Not a blocker: I don't think we've used `Text` from polaris-react yet in our site components and I'm wondering if this could have an unintended ripple effect 🤔 I'm not against using it in the site, especially since we're planning on using the polaris gen3 typescale in the site anyway. I think if we use `Text` here we should make plans to use `Text` in our other components where possible

Reply:
I opted to use the text component here because the Figma designs were had variant mappings.
If we prefer to not, let me know and I can refactor this in a follow up pr 👍